spec/unit/webrtc/groupCall.spec.ts

```diff
@@ -280,6 +289,143 @@ describe('Group Call', function() {
             }
         });
 
+        describe("call feeds changing", () => {
+            let call: MockCall;
+            const currentFeed = new MockCallFeed(FAKE_USER_ID_1);
+            const newFeed = new MockCallFeed(FAKE_USER_ID_1);
+
+            beforeEach(async () => {
+                jest.spyOn(groupCall, "getUserMediaFeedByUserId");
+                // @ts-ignore Mock
+                jest.spyOn(groupCall, "addUserMediaFeed");
+                // @ts-ignore Mock
+                jest.spyOn(groupCall, "replaceUserMediaFeed");
+                // @ts-ignore Mock
+                jest.spyOn(groupCall, "removeUserMediaFeed");
+                // @ts-ignore Mock
+                jest.spyOn(groupCall, "addScreenshareFeed");
+                // @ts-ignore Mock
+                jest.spyOn(groupCall, "replaceScreenshareFeed");
+                // @ts-ignore Mock
+                jest.spyOn(groupCall, "removeScreenshareFeed");
+
+                jest.spyOn(currentFeed, "dispose");
+                jest.spyOn(newFeed, "measureVolumeActivity");
+
+                jest.spyOn(groupCall, "emit");
+
+                call = new MockCall(room.roomId, groupCall.groupCallId);
+
+                await groupCall.create();
+            });
+
+            it("ignores changes, if we can't get user id of opponent", async () => {
+                const call = new MockCall(room.roomId, groupCall.groupCallId);
+                call.getOpponentMember = (() => {}) as unknown as any;
+
+                // @ts-ignore Mock
+                expect(() => groupCall.onCallFeedsChanged(call)).toThrowError();
```

Comment:
What is the `@ts-ignore` ignoring? If it's because `onCallFeedsChanged` is a private method, is there any reasonable way to test it without calling it directly?

Reply:
While this can be done, I am not sure it is worth the work - it would require making `MockCall` an `EventTarget` among other things

Reply:
OK, that's fine then